src/UI/NavigationContainer.elm

```diff
@@ -39,40 +97,70 @@ import UI.Utils.Element as Element
 -- Expose
 
 
+{-| This record must be generated with [`Nav.menuPage`](UI-NavigationContainer#menuPage)
```

Comment:
Ok, so maybe if we want to make it accessible only using `Nav.menuPage` we must make this an opaque type ( I think it's a good idea).

Same for `MenuAction`